src/devices/danfoss.ts

```diff
@@ -139,7 +139,8 @@ const definitions: DefinitionWithExtend[] = [
                             'property *at most* every 5 minutes or every 0.1K change in measured room temperature.',
                     )
                     .withValueMin(-8000)
-                    .withValueMax(3500),
+                    .withValueMax(3500)
+                    .withUnit('0.01 °C'),
```

Comment:
This is not a valid unit, must be e.g. `°C`, maybe use that and in the description clarify it is `0.01 °C` to prevent a breaking change?

Reply:
In that case I think it's better to leave the formal unit unspecified and just use a description on how the value is interpreted as I originally did.